lib/WaifuPics.js

```diff
@@ -27,14 +34,20 @@ const WaifuPics = {
    */
   endpoints () {
     return request('GET', '/endpoints')
-  }
+  },
+
+  many: {}
 }
 
 Object.keys(endpoints).forEach((type) => {
   WaifuPics[type] = {}
+  WaifuPics.many[type] = {}
 
   endpoints[type].forEach((category) => {
-    WaifuPics[type][category] = () => request(`/${type}/${category}`)
+    const endpoint = `/${type}/${category}`
+
+    WaifuPics[type][category] = () => request('GET', endpoint)
+    WaifuPics.many[type][category] = (exclude) => request('POST', '/many' + endpoint, { body: JSON.stringify({ exclude }) })
   })
```

Comment:
This is the best so far, maybe it will improve further.

Reply:
I don't know if exclude images is effective, I tried to eliminate a large scale of images to see where we were going but it seems to be large enough to always return others